pvlib/shading.py

```diff
@@ -342,3 +342,213 @@ def projected_solar_zenith_angle(solar_zenith, solar_azimuth,
     # Eq. (5); angle between sun's beam and surface
     theta_T = np.degrees(np.arctan2(sx_prime, sz_prime))
     return theta_T
+
+
+def shaded_fraction1d(
+    solar_zenith,
+    solar_azimuth,
+    axis_azimuth,
+    shaded_tracker_rotation,
+    *,
+    collector_width,
+    pitch,
+    axis_tilt=0,
+    surface_to_axis_offset=0,
+    cross_axis_slope=0,
+    shading_tracker_rotation=None,
+):
+    r"""
+    Shaded fraction in the vertical dimension of tilted rows, or perpendicular
+    to the axis of horizontal rows.
+
+    If ``shading_tracker_rotation`` isn't provided, assumes both the shaded
+    row and the one blocking the direct beam
+    share the same rotation and azimuth values.
+
+    .. warning::
+        This function assumes the roles of the shaded and shading trackers are
+        the same during all the day. If the trackers allow for different
+        shading or shaded roles, e.g. a N-S single-axis tracker, you must
+        switch the inputs depending on the sign of the projected solar zenith
+        angle. See :ref:`ns_sat_case`.
+
+    .. versionadded:: 0.10.5
+
+    Parameters
+    ----------
+    solar_zenith : numeric
+        Solar position zenith, in degrees.
+    solar_azimuth : numeric
+        Solar position azimuth, in degrees.
+    axis_azimuth : numeric
+        In degrees. North=0º, South=180º, East=90º, West=270º.
+    shaded_tracker_rotation : numeric
```

Comment:
I know we have gone back and forth with these parameter names previously, but a new option occurs to me: `shaded_row_rotation`.  The advantage is that `row` is generic and applies equally to tracking and fixed-tilt arrays.  

@echedey-ls what do you think?

Reply:
> I know we have gone back and forth with these parameter names previously

No worries, I rather address these improvements now than seeing that comment after the merge. 

> The advantage is that row is generic and applies equally to tracking and fixed-tilt arrays.

I also 100% agree on that.